docs/source/conf.py

```diff
@@ -44,14 +44,30 @@
 
 #
 #
-from mhkit import __version__ 
 import sphinx_rtd_theme
 import os
 import sys
 #
+# this_dir = os.path.dirname(os.path.abspath(__file__))
+# matlab_src_dir = os.path.abspath(os.path.join(this_dir, '../..'))
+# primary_domain = 'mat'
+
+#Kelley previous
 this_dir = os.path.dirname(os.path.abspath(__file__))
 matlab_src_dir = os.path.abspath( '../../MHKiT-MATLAB/')
 sys.path.insert(0, matlab_src_dir)
+# Kelely probably don't need
+#python_src_dir = os.path.abspath( '../../MHKiT-Python/')
+#sys.path.insert(0, python_src_dir)
+
+# from rebeccca
+# this_dir = os.path.dirname(os.path.abspath(__file__))
+# matlab_src_dir = os.path.abspath(os.path.join(this_dir, '..'))
+# primary_domain = 'mat'
+
+#
```

Comment:
Clean up these comments. Remove the unnecessary parts.

Reply:
I will remove unnecessary comments. 

This is the MATLAB path configuration, pointing to the MHKiT-MATLAB submodule. It took some trial/error to make sure the API documentation was compiling properly by pointing to the MHKiT-MATLAB submodule, and not a previously installed version of MHKiT-MATLAB or MHKiT-Python. This is something we should review as a team to make sure the submodules working for everyone.